docs/command-line-tutorial.md

```diff
@@ -0,0 +1,324 @@
+---
+sidebar_position: 3
+---
+
+# Command Line Tutorial
+
+Learn Divvi Up via a local instance of all Divvi Up components running via
+`docker compose`.
+
+Following this tutorial will:
+
+- Install required software and tools
+- Run Divvi Up and a pair of aggregators locally in Docker
+- Shard some measurements into reports and upload them, simulating DAP clients
+- Collect the results of an aggregation
+
+If any of these terms are unfamiliar please read the
+[How it Works](./how-it-works) doc.
+
+## Requirements
+
+To get started, you'll need:
+
+- A Unix like shell on macOS, Linux or Windows
+- A `divviup` binary (see instructions below)
+- A `compose.yaml` (see instructions below)
+- [Docker](https://docs.docker.com/get-docker/) and
+  [Docker Compose](https://docs.docker.com/compose/install/).
+
+### Install `divviup`
+
+`divviup` is a command line (CLI) tool for doing all basic operations on both
+the Divvi Up API and Distributed Aggregation Protocol (DAP) API endpoints. It's
+only likely to work if the leader aggregator is
+[Janus](https://github.com/divviup/janus). See `divviup --help` for details on
+all of the commands.
```

Comment:
The main possible incompatibility is the choice of authentication mechanism. (this might be good to call out here) The DAP client subcommand should work with most any DAP aggregators, for example, so long as no client authentication DAP extension is needed. The DAP collector subcommand may be hit-or-miss, because it only uses our `Authorization: Bearer ...` scheme. If Daphne were the leader aggregator, for example, then this wouldn't work, as Daphne just supports `DAP-Auth-Token: ...` and mTLS.